### ISSUE TITLE
[OTHER] small color changes for accessibility

### ISSUE THREAD

**Issue by lovelacecoding** [Njong392/Abbreve#71]
### What would you like to share?

Hi! I have some small color changes that can make the application more accessible.

original:
![image](https://user-images.githubusercontent.com/92865452/194026175-eb32d6ce-324a-4672-a20a-4d7b62683da8.png)
new version:
![image](https://user-images.githubusercontent.com/92865452/194026418-a72b0ae2-cac6-444b-a18f-ea45c304f8dc.png)

I changed the original purple color to deep purple and added a slightly lighter purple to make some parts of the text more readable. I also made the gray a bit lighter. 

Let me know if you like it!

**Comment by ctoffanin** [comment]
I agree with you. I like it!

**Comment by Njong392** [comment]
Yes Lou! Looking at it now, I see how cool the contrast is. You can go ahead and submit a pull request for this!
